README.md

```diff
@@ -1,24 +1,126 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Blog App <a name="about-project"></a>
 
-* System dependencies
+**Budget-app** Budget app built with Ruby on Rails. You can create categories and have a list of transactions associated with a category, you can add new transactions and see the money you spent in each category and transaction.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+Ruby on Rails, Visual Studia Code, WSL
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+- **Ruby on Rails**
 
-* Services (job queues, cache servers, search engines, etc.)
+- **WSL**
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+> - List Categories
+> - Add a new Category
+> - List Transactions
+> - Add a new Transaction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+## 🚀 Live Demo <a name="live-demo"></a> 
+
+
+
+- [Live Demo Link](https://budget-app-1-l7xh.onrender.com)
+- [Presentation Video](https://www.loom.com/share/3a8e390b4689472d8f16424b4a5f065c)
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=the%20project%20features.-,you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.,-For%20recording%20the), please highlight some interesting piece of code or something you built that you are very proud of.